backend/src/repository/analysis_collection.py

```diff
@@ -131,11 +131,27 @@ def find_file_by_name(self, analysis_name: str, file_name: str):
             return None
 
         for file in analysis['supporting_evidence_files']:
-            if file['filename'] == file_name:
+            if file['name'] == file_name:
                 return file
 
         return None
 
+    def file_exists_in_analysis(self, analysis_name: str, file_name: str):
```

Comment:
This function is nearly identical to the one above it. The only difference is that this one returns `True` on success or `False` while the other function returns the file meta data on success and `None` on failure.

I would argue that we use the existing function and check to see if the result from `find_file_by_name` is not `None`

![Screenshot 2023-05-15 at 11 36 30 AM](https://github.com/uab-cgds-worthey/rosalution/assets/5799712/0b24639b-0b3a-4076-ac68-233d8c5d8707)